client/reader/liked-stream/empty.jsx

```diff
@@ -36,7 +36,7 @@ class TagEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read"
 				>
-					{ this.props.translate( 'Back to Following' ) }
+					{ this.props.translate( 'Back to following' ) }
```

Comment:
Should we keep "Following" in title case? Seems like a feature/section name that lists followed blogs.

Reply:
Good catch here @mmtr. You're right, "Following" in this case in the feature/section name.